keyboards/tweetydabird/lotus58/info.json

```
@@ -0,0 +1,150 @@
+{
+    "manufacturer": "Tweetys Wild Thinking",
+    "keyboard_name": "Lotus 58 Glow (QMK)",
+    "maintainer": "TweetyDaBird",
+    "bootloader_instructions": "Short marked pads on PCB, or hold top-outer key when plugging in each hand.",
+    "build": {
+        "lto": true
+    },
+   "diode_direction": "COL2ROW",
+    "encoder": {
+        "rotary": [
+            {
+                "pin_a": "F5",
+                "pin_b": "F4",
+                "resolution": 2
+            }
+        ]
+    },
+    "features": {
+        "audio": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "grave_esc": false,
+        "magic": false,
+        "mousekey": false,
+        "nkro": true,
+        "oled": true,
+        "rgblight": true,
+        "space_cadet": false,
+        "split": true,
+        "tap_dance": false,
+		"tri_layer": true
```

```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "encoder": true,
        "extrakey": true,
        "grave_esc": false,
        "magic": false,
        "mousekey": true,
        "nkro": true,
        "oled": true,
        "rgblight": true,
        "space_cadet": false,
        "split": true,
		"tri_layer": true
```

Space permitting, the feature-set should be more "default" like

```suggestion
        "bootmagic": true,
        "command": false,
        "console": false,
        "encoder": true,
        "extrakey": true,
        "mousekey": true,
        "nkro": true,
        "oled": true,
        "rgblight": true,
        "split": true,
		"tri_layer": true
```